### PR TITLE
Init

### PR DESCRIPTION
**Pull request by sbs2001**

### CODE REVIEW

File: pkg/cf/cf.go [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+				return err
+			}
+			m.logger.Tracef("batch %d delete key resp: %+v", batch, resp)
+			return nil
+		})
+	}
+	if err := deleterGrp.Wait(); err != nil {
+		return err
+	}
+	m.logger.Info("Done deleting keys")
```
Comment (AlteredCoder) [inline pkg/cf/cf.go:348]: can we log how many decisions we deleted ? and have a better log eg "%d decisions have been deleted"

File: pkg/cf/cf.go [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+				}
+				m.logger.Tracef("batch %d write key resp: %+v", batch, resp)
+				return nil
+			})
+		}
+		if err := writerErrGroup.Wait(); err != nil {
+			return err
+		}
+		m.KVPairByDecisionValue = newKVPairByValue
+		m.logger.Info("Done adding decisions")
```
Comment (AlteredCoder) [inline pkg/cf/cf.go:440]: same here

**Review by AlteredCoder — CHANGES_REQUESTED** [review]

**Review by AlteredCoder — COMMENTED** [review]
- Can we name file/folder `cloudflare` instead of `cf`
- Please add more comments in cf.go to explain what the functions do

File: pkg/cf/cf.go (hunk 2 above)
Comment (sbs2001) [reply pkg/cf/cf.go:440]: Done

**Comment by sbs2001** [comment]
@AlteredCoder  done via https://github.com/crowdsecurity/cs-cloudflare-worker-bouncer/pull/1/commits/c5cd555282888ba6cadd3cdfd7268097c7ffa303

**Comment by mmetc** [comment]
Several configuration options take the name from the legacy cloudflare bouncer, which sets them apart from the other bouncers. Example: crowdsec_lapi_url, log_mode vs log_media etc.

It would be nice to have some consistency and predictability -- for ease of use and to minimize differences in the build/install/configuration scripts

**Comment by sbs2001** [comment]
@mmetc thanks for the review. I've updated the config to be same as cs-blocklist-mirror

File: scripts/_bouncer.sh [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    # assuming it is running on the same host as the
+    # bouncer.
+    varname=$1
+    if [ "$varname" = "" ]; then
+        msg err "missing required variable name"
+        exit 1
+    fi
+    command -v cscli >/dev/null || return 0
+
+    port=$(cscli config show --key "Config.API.Server.ListenURI" 2>/dev/null | cut -d ":" -f2 || true)
```
Comment (mmetc) [inline scripts/_bouncer.sh:186]: please force raw output of cscli, see https://github.com/crowdsecurity/cs-custom-bouncer/pull/91/commits/e30ff3fbf93d037ff7a649feae1d7a2c2c96eaf1

Comment (sbs2001) [reply scripts/_bouncer.sh]: fixed via https://github.com/crowdsecurity/cs-cloudflare-worker-bouncer/pull/1/commits/b5850a6415bbdc35dcd039b51cbc8ab55d7bf409

**Review by rr404 — APPROVED** [review]
LGTM for init PR
We'll package this for a beta test by Tobias
We'll make further changes based on his comments if necessary

**Pull request merged**